.github/workflows/README.md

```diff
@@ -0,0 +1,126 @@
+# AlphaX Release Process
+
+## Release a new version
+### Tagging of changes
+In order to have release notes automatically generated, changes need to be tagged with labels.
+The following labels are used (should be safe-explanatory):
+`breaking-change`, `bug`, `enhancement`.
+
+### Release a new version
+Note: Releases need to be done from the `main` branch.
+
+1. Bump the version locally to (e.g. to `X.Y.Z`) and merge the change to `main`.
+2. Create a new draft release on GitHub using the
+[Create Draft Release](https://github.com/MannLabs/alphashared/.github/workflows/create_release.yml) workflow.
+You need to specify the commit to release, and the release tag (e.g. `vX.Y.Z`).
+3. Test the release manually.
+4. Add release notes and publish the release on GitHub.
+5. Run the [Publish on PyPi](https://github.com/MannLabs/alphashared/.github/workflows/publish_on_pypi.yml) workflow,
+specifying the release tag (e.g. `vX.Y.Z`).
+6. (optional, if present) Run the [Publish Docker Image](https://github.com/MannLabs/alphashared/.github/workflows/publish_docker_image.yml) workflow,
+specifying the release tag (e.g. `vX.Y.Z`).
+
+
+## Installation of the release pipeline
+### Requirements on the caller repository
+In order to be compatible with the workflow, the calling repository must meet the following requirements:
+
+#### Version
+The file `<package_name>/__init__.py` must contain the following line:
+
+```python
+__version__ = "<version to release, e.g. 1.2.3>"
```

Comment:
where is the version to release specified? Isn't the version specified here by definition the version to release?

Reply:
nope, it's taken from here